docs/telemetry.md

```diff
@@ -12,16 +12,4 @@ Fenix creates and tries to send a "baseline" ping. It is defined inside the [`me
 
 ## Events
 
-Fenix sends event pings that allows us to measure feature performance.
-
-| Event           | Glean Key         | Leanplum Key | extras                |
-|-----------------|-------------------|--------------|-----------------------|
-| OpenedApp       | app_opened        | E_Opened_App | source*               |
-| SearchBarTapped | search_bar_tapped |              | source**              |
-| EnteredUrl      | entered_url       |              | autocomplete***       |
-| PerformedSearch | performed_search  |              | search_suggestion**** |
-
-* `source`: The method used to open Fenix (For exmaple: `app_icon` or `link`)
-** `source`: The view the user was on when they initiated the search (For example: `Home` or `Browser`)
-*** `autocomplete`: A boolean that tells us wether the URL was autofilled by an Autocomplete suggestion
-**** `search_suggestion`: A boolean that tells us wether or not the search term was suggested by the Awesomebar
\ No newline at end of file
+Fenix sends event pings that allows us to measure feature performance. These are defined inside the It is defined inside the [`metrics.yaml`](https://github.com/mozilla-mobile/fenix/blob/master/app/metrics.yaml) file.
```

Comment:
Nit: typo in this sentence

Also, you should keep the app_opened ping because the Leanplum key is not documented in Glean I assume, and #981 is still open.